examples/Links/ExternalLink.md

```diff
@@ -0,0 +1,12 @@
+This component is a styled version of a regular `<a>` link, for use when we need to link to external resources.
+
+```jsx
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSignOutAlt } from '@fortawesome/free-solid-svg-icons';
+
+<ExternalLink href="https://key.harvard.edu/logout">
+  Log Out
+  {' '}
```

Comment:
Originally I was thinking that maybe the space could be integrated into the component definition, but this seems to be a rare enough case to be using both text and an icon. Also, I noticed that when you hover, the underline extends a little past the text due to the space, but this is a very tiny detail - 
<img width="295" alt="Screen Shot 2020-10-15 at 2 51 51 PM" src="https://user-images.githubusercontent.com/29589689/96174244-047ddb00-0ef7-11eb-918d-659074514683.png">

Reply:
That is weird. I suspect there's something in the FontAwesome library that's removing the underline. In any case, I'd prefer to leave handling of the link content to the developer, otherwise we'll drive ourselves crazy trying to handle edge cases.

Re: link color, I believe we had discussed this with Vittorio and decided to use a consistent color for all of links, so I agree that we wouldn't want to provide other color options.